crew

```diff
@@ -166,6 +189,9 @@ def help (pkgName)
     puts "If `-k` or `--keep` is present, the `CREW_BREW_DIR` (#{CREW_BREW_DIR}) directory will remain."
     puts "If `-s` or `--build-from-source` is present, the package(s) will be compiled instead of installed via binary."
     puts "If `-v` or `--verbose` is present, extra information will be displayed."
+  when "list"
+    puts "List available or installed packages"
+    puts "Usage: crew list (available|installed)"
```

Comment:
Can we use brackets instead of parenthesis?
```
Usage: crew list [available|installed]
```

Reply:
Based on the [docopt](http://docopt.org/) documentation brackets mean optional elements.

Since using `crew list` only doesn't really make sense by itself, I think that having one of the 2 option a requirement is better

Reply:
Understood but don't we want a default option if no parameters are supplied?  In that case, it would be optional.  Just sayin'.

Reply:
It really depends if we want a default option. I think that the command `crew list` doesn't say anything. 

Same thing with the `crew files` command, it doesn't have a default and without an argument the command doesn't make sense

Reply:
I suppose if we defaulted to list all the packages it would essentially be the same as `crew search` with no parameters.  I'll go ahead and merge this and we can update later if enough users feel it makes sense to provide a default option.

Reply:
Sound good for me, thank you very much.